src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateHistory = [];
+  const stateNewObject = Object.assign({}, state);
+
+  // stateHistory.push(state);
```

Comment:
Consider uncommenting this line if the initial state should be included in the `stateHistory`. This will ensure that the history starts with the original state before any actions are applied.